tests/console/consoletest_finish.pm

```diff
@@ -26,6 +26,8 @@ sub run {
     save_screenshot();
 
     systemctl 'unmask packagekit.service';
+    # unmask serial-getty service which pollutes serial output
+    systemctl 'unmask serial-getty@' . get_var('SERIALDEV') ? get_var('SERIALDEV') : 'ttyS0';
```

Comment:
Just keep it disabled. Quite some test modules might use script_output after this test module and I am not aware of any use case where we need the getty

Reply:
I decided to play it safe, but I guess it won't harm if we keep it masked.